go/x509/verify_test.go

```diff
@@ -83,6 +84,30 @@ var verifyTests = []verifyTest{
 		errorCallback: expectExpired,
 	},
 	{
+		leaf:              googleLeaf,
+		intermediates:     []string{thawteIntermediate},
+		roots:             []string{verisignRoot},
+		currentTime:       1,
+		dnsName:           "www.google.com",
+		disableTimeChecks: true,
+
+		expectedChains: [][]string{
+			{"Google", "Thawte", "VeriSign"},
+		},
+	},
+	{
+		leaf:              googleLeaf,
+		intermediates:     []string{thawteIntermediate},
+		roots:             []string{verisignRoot},
+		currentTime:       10000000000,
+		dnsName:           "www.google.com",
+		disableTimeChecks: true,
+
+		expectedChains: [][]string{
+			{"Google", "Thawte", "VeriSign"},
```

Comment:
disableTimeChecks defaults to false, so every test that does not specify it to be true is testing it set to false.  I believe there is one test that checks for an expiration error if currentTime = 1, but you're right that it could also do with a similar test for currentTime set to way in the future.